frontend/packages/operator-lifecycle-manager/src/components/create-operand.tsx

```diff
@@ -253,8 +253,8 @@ export const CreateOperandForm: React.FC<CreateOperandFormProps> = ({
           }
           const schemaPath = field.path.split('.').join('.properties.');
           const required = (_.get(
-            openAPI,
-            _.dropRight(['properties', 'spec', ...field.path.split('.')])
+            openAPI.properties,
```

Comment:
Will `openAPI` ever be null? We might want

```suggestion
            openAPI?.properties,
```

Reply:
@spadgett Thanks Sam! It is a good suggestion.